### PR TITLE
add version number and add files

### PR DESCRIPTION
**Pull request by pinkywafer**
resources.json (to allow downloading of additional files)
and __init__.py for HA

### CODE REVIEW

File: custom_components/owlintuition/sensor.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
 import homeassistant.helpers.config_validation as cv
 from homeassistant.util import Throttle
 
+VERSION = '0.0.2'
```
Comment (glpatcern) [inline custom_components/owlintuition/sensor.py]: The current version of this component is actually `1.3.0`.

File: resources.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[
+  "https://raw.githubusercontent.com/custom-components/sensor.owlintuition/master/custom_components/owlintuition/manifest.json",
+  "https://raw.githubusercontent.com/custom-components/sensor.owlintuition/master/custom_components/owlintuition/sample.owl2.xml",
```
Comment (glpatcern) [inline resources.json]: I have included this file as a form of documentation, but it is not needed to run the component. Thus I'd remove it from here.

**Comment by pinkywafer** [comment]
Updated as requested

**Comment by glpatcern** [comment]
Thanks, OK let's go with version 1.4: I've just pushed the relevant changes on the `owl.json` and `README.md` files as well.

**Pull request merged**